src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super.setColor(color);
```

Comment:
use constructor `super(color);` instead of setter here. fix the same in all figures classes.